src/api/export.ts

```diff
@@ -201,7 +212,7 @@ async function _export({
 			type = 'text/html';
 			body = `<script>window.location.href = "${location.replace(origin, '')}"</script>`;
 
-			tasks.push(handle(resolve(root.href, location)));
+			tasks.push(handle(resolve(entryPoints[currentRoot].href, location)));
```

Comment:
What's the reason for resolving this relative to the current entry point? I _think_ this should still always be relative to `root`. My understanding is that the multiple entry points is just so that the crawler can find pages it wouldn't ordinarily find. All of them should still have the same base href value.

Reply:
The test you wrote still passes when using `root.href`, fwiw

Reply:
Yeah, I think you're right. I don't think I looked too deeply into this but I guess this clause is handling redirects/ 300s? I think it's using the `Location` header and building a new url to `handle` from that. My tests definitely don't test this behaviour, I don't know if any of the existing ones do either but resolving with anything but the root url would fail, I think.

Reply:
Yeah, looking back this clears up some of the confusion I had at the time with the whole `entryPoints[currentRoot]` thing, I wondered why that was necessary. This correction removes the need for that entirely. I probably should have taken more time to understand the code before implementing this.